### PR TITLE
Update Export Paths

### LINKED ISSUE #62: Exports missing request params
The order export endpoint docs specifically are missing any information about request parameters.

While correcting that, let's also verify that all other export endpoint docs include request params as needed.

### PR DESCRIPTION
**Pull request by nickcoleman**
Update Export Paths with missing or incorrect request parameters

fixes #62

### CODE REVIEW

File: _ordering/get_orders_export.md [hunk 1]
```diff
@@ -3,29 +3,20 @@ title: /orders/export/
 name: Get Orders Export
 position: 3.01
 method: post
-description: Get an Export of the Orders via an email
+description: Get an Export of the Order List via an email
```
Comment (baileymatthewr) [inline _ordering/get_orders_export.md]: Jason asked me to specifically not use order lists. That was how I originally had this. You need to use the normal plural, orders.

File: _ordering/get_orders_export.md [hunk 2]
```diff
@@ -35,82 +26,42 @@ Get an Export of the Orders via an email to your email address on file. This API
...
 
-| Code | Name                   | Meaning                                                                      |
-|------|-------------------------------------------------------------------------------------------------------|
-| 200  | OK                     | The API call was received and response is provided                           |
-| 400  | Bad Request            | Generally, something required for the request is missing                     |
-| 401  | Unauthorized           | Generally, the username or password is incorrect                             |
-| 403  | Permission Denied      | Generally, the user does not have permission to perform the requested action |
-| 404  | Not Found              | Generally, the call is not sent to the correct URL                           |
-| 415  | Unsupported Media Type | Generally, this is a syntax problem                                          |
+Expected responses include 200, 400, 401, 403, or 404.                                     |
```
Comment (baileymatthewr) [inline _ordering/get_orders_export.md]: Very nice!

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+      "workbench.colorTheme": "Nord",
```
Comment (Gidgidonihah) [inline .vscode/settings.json]: kill this file.

File: _ordering/get_orders_export.md [hunk 4]
```diff
@@ -35,82 +26,42 @@ Get an Export of the Orders via an email to your email address on file. This API
...
-line_item_status
-: (array) One or more line item statuses. Will include orders with statuses based upon that `status_conjunction`. Possible Values are: `unallocated`, `allocated`, `rejected`, `has_tracking`, `backordered`
-
-status_conjunction
-: (string) Indicates how the line_item_status relates to the order response. Possible values are: `and`, `or`, `all`. Default: `or`. `And` will return all orders that have at least 1 line item matching each supplied status. `or` will return all orders that have at least one line item matching any of the supplied statuses. `only` will return orders that have ALL line items matching the supplied status. Only one line_item_status should be supplied when using `only`.
+#### Optional:
 
-start_date
-: (string) The Start Date for your search results. The date must be written in the following format "YYYY-MM-DDThh:mm:ss.000Z"
+none
```
Comment (Gidgidonihah) [inline _ordering/get_orders_export.md]: No need for optional if there are none.

File: _ordering/get_order_detail_export.md [hunk 5]
```diff
@@ -1,143 +0,0 @@
----
-title: /orders/&ltorder_uuid&gt/export/
```
Comment (Gidgidonihah) [inline _ordering/get_order_detail_export.md]: Why did this get removed?

File: _ordering/get_orders_export.md (hunk 1 above)
Comment (Gidgidonihah) [reply _ordering/get_orders_export.md]: good catch

File: _ordering/get_order_detail_export.md (hunk 5 above)
Comment (nickcoleman) [reply _ordering/get_order_detail_export.md]: Can we discuss ... maybe I'm missing something

File: _ordering/get_orders_export.md (hunk 4 above)
Comment (nickcoleman) [reply _ordering/get_orders_export.md]: K

File: _ordering/get_order_detail_export.md (hunk 5 above)
Comment (nickcoleman) [reply _ordering/get_order_detail_export.md]: Per conversation ... isn't there anymore

**Pull request merged**